_locales/en/messages.json

```diff
@@ -114,7 +114,7 @@
     "message": "Website Themes"
   },
   "popupFeatures": {
-    "message": "Extension Popup Features"
+    "message": "Extension Features"
```

Comment:
I like the idea of this because the one addon that runs in the background, Scratch Notifier, in addition to all of the popup addons, perfectly fits in, but I wonder if there's a clearer way of putting this. Technically one could say that everything is an "extension feature", and "Extension Popup Features" was more self-explanatory about how those kinds of addons work.

But like @BroJac5246 said, if you think about it, if each category is a subset of all of Scratch Addons' addons and there's already an "All" tab, then logically this category won't contain every feature and therefore must be about something else (in this case, it's related to the non-website/editor features - those that are part of the extension's inner workings and the popup), so I think this works. I'd just say that if Scratch Notifier ever disappears, we could change this back to "Extension Popup Features".

Reply:
@DNin01 did you mean to tag me? (Apparently the GitHub app's notifications aren't working for me so sorry for the late reply)

Reply:
I agree that referring to the popup directly isn't technically the most accurate, however "extension features" sounds way too ambiguous. As pointed out, everything, including all of the addons, are features of the extension. Also, many users won't determine the difference between "addon" and "extension," especially given that the two are pretty much identical, just one is browser-level while one is extension-level.

I would change it back unless a better wording can be found, which would be ideal. "Extension/Scratch Addons internal addons" or something similar might work, but what do you think?

Reply:
Would "Browser features" be any less confusing?

Reply:
> Would "Browser features" be any less confusing?

No because they're still not the browser's features.

Reply:
The only othee things I can think of are "Background features" and "Other features" which is already used to categorize non-featured addons. Your suggestion describes it well but I think it's a little too long.

Reply:
> "Extension/Scratch Addons internal addons"

The word "internal" makes it sound like these addons aren't intended for the user.

Reply:
A few alternative options:
- Separate the categories into two groups, "Scratch website" and "Scratch Addons", or something like that
- Don't list extension popup features as addons. Instead, make them toggleable through More Settings. I would imagine most people would want to keep these enabled anyway.
  - Scratch Notifier could then stay in Website Features or in some other category, potentially "Miscellaneous"/"Other".
  - And you'd probably configure what games display on the Cloud Games addon through the popup.
  - Wonder if the message count badge (or even Scratch Notifier) could be turned on/off from the Scratch Messaging addon in addition to More Settings, to make it easier to disable for those who don't need it?

Reply:
> > "Extension/Scratch Addons internal addons"
> 
> The word "internal" makes it sound like these addons aren't intended for the user.

> The only othee things I can think of are "Background features" and "Other features" which is already used to categorize non-featured addons. Your suggestion describes it well but I think it's a little too long.

These might make sense together, by replacing "internal" with "background"... though "Scratch Addons background addons" sounds a little weird.

Reply:
Would just "Background addons" be enough?

Reply:
> Would just "Background addons" be enough?

That doesn't sound right for popup features.

Reply:
As far as I remember most of what you said I had as internal dialogue in my head when trying to name the category.
I would suggest against trying to propose changes to this category name unless it's the only change in the PR.

Reply:
Should I undo the Scratch notifier category change then?

Reply:
Yeah, probably the best thing to do right now would be to separate the category name change from this PR. Maybe even open an issue or discussion for it first.